src/main/java/Command.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Command class.
+ */
+public abstract class Command {
+    private String inputCommand; //input command by the user
+    private boolean isExit;
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public Command(String inputCommand, boolean isExit) {
+        this.inputCommand = inputCommand;
+        this.isExit = isExit;
+    }
+
+    /**
+     * Return the command inputted by user.
+     * @return
+     */
+    public String getInputCommand() {
+        return this.inputCommand;
+    }
+
+    /**
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). There should also be an empty line between description and parameter section.